### PR TITLE
konoha: style cleanup

### PR DESCRIPTION
**Pull request by DomT4**
Mostly a test of the special dependency problem we've been seeing in places, but I'll merge the style changes w/bottles if it actually passes.

### CODE REVIEW

**Comment by mikemcquaid** [comment]
:+1:

**Comment by mikemcquaid** [comment]
Looks like the tests are failing.

**Comment by mikemcquaid** [comment]
The in-`install` tests, that is.

File: Library/Formula/konoha.rb [hunk 1]
```diff
@@ -14,14 +15,13 @@ class Konoha < Formula
...
+  depends_on :python if MacOS.version <= :snow_leopard # for python glue code
 
   def install
-    args = std_cmake_args + [".."]
     mkdir "build" do
-      system "cmake", *args
+      system "cmake", "..", *std_cmake_args
       system "make"
-      system "make", "test" if build.include? "tests"
+      system "make", "test" if build.with?("tests") || build.bottle?
```
Comment (xu-cheng) [inline Library/Formula/konoha.rb]: I find behavior caused by `build.with?("tests") || build.bottle?` can be confusing without reading the code. Why not just enable test by default?

Comment (xu-cheng) [inline Library/Formula/konoha.rb]: In more general point, I think we should always enable test by default. The only exception I can think of is that test is buggy or take enormous long time and it's not security or crypto formula.

**Comment by DomT4** [comment]
> Looks like the tests are failing.

Yeah. I did a bad thing and didn't test the PR before I pushed it because I expected it to die on the special dep and for us to deal with that. As it happened, it passed the special dep and then failed the `make check` :sweat:.

File: Library/Formula/konoha.rb (hunk 1 above)
Comment (DomT4) [inline Library/Formula/konoha.rb]: I think all bottled builds should have test/check done on them, but I think it has more use to packagers than end users. Not against making it default all over the place barring special cases, but I think it's more useful to us than most people.

**Comment by mikemcquaid** [comment]
> Yeah. I did a bad thing and didn't test the PR before I pushed it because I expected it to die on the special dep and for us to deal with that. As it happened, it passed the special dep and then failed the make check .

That's OK to do! That's what CI is for!

**Comment by DomT4** [comment]
I normally test them locally, but really wasn't expecting this to get that far into the build so didn't bother, heh. Slightly annoyed it *did* get that far into the build because the special dep issue remains elusive.

**Comment by apjanke** [comment]
It's passing now. Is this good to merge?

**Comment by mikemcquaid** [comment]
:+1:

**Comment by DomT4** [comment]
Heh, reported the failure upstream the other night, wrote a working test, pushed the branch again to trigger a new build... and then forgot all about the PR :sweat:. Merged.

**Pull request closed without merging**